tools/build.sh

```diff
@@ -173,21 +160,21 @@ LIST_CSS_TOPLEVEL="../../../node_modules/@nimiq/style/nimiq-style.min.css ../../
 
 # generate bundle files
 output "📦  Generating bundle files"
+# put config first
+cat src/config/config.$BUILD.js >> dist/request/$JS_COMMON_BUNDLE
```

Comment:
The config depends on a constant from `Constants.js`. Have you tested, if this throws a `Constants is undefined` error for the built files?

Reply:
> It only resolves #258 to a degree, as the default config for local is still `'*'`. I am aware that it's necessary for IP-based testing, but it does not _really_ fix the issue.

I don't think the issue was local config. Might be problematic for an offline build, but when we add an option for that we might as well restrict the default local config to be not compatible with that.

Reply:
...I tested first and replaced the values by constants later.
Ok. this needs a fix.